pinot-core/src/test/java/org/apache/pinot/queries/TimestampQueriesTest.java

```diff
@@ -0,0 +1,229 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.queries;
+
+import java.io.File;
+import java.io.IOException;
+import java.sql.Timestamp;
```

Comment:
It looks like we should use java.time.*

This article says joda time, java.sql.time* and all of old time/date classes usage should be migrated to use java.time

https://softwareengineering.stackexchange.com/questions/190891/joda-time-vs-java-time

Reply:
`Both Joda-Time and the original Java date-time classes are now outmoded by the java.time classes built into Java 8, Java 9, and later.` from the thread

Reply:
Here we should stick with SQL timestamp format. The `java.sql.Timestamp` is used for ser/de only. Internally we can convert it to `java.time` classes for calculation.